build-system/tasks/visual-diff/index.js

```diff
@@ -805,10 +805,6 @@ async function ensureOrBuildAmpRuntimeInTestMode_() {
  * @return {!Promise<!puppeteer.BrowserFetcher>}
  */
 async function installDependencies_() {
```

Comment:
This function can now be replaced by just directly importing these packages at the top, right?

Reply:
They cannot because when we load this file in `--help` mode, the dependencies are not going to be present. I've updated the JSDoc here as well.